charts/splunk-connect-for-snmp/templates/_helpers.tpl

```diff
@@ -14,6 +14,85 @@
 {{- printf "redis://%s-redis-headless:6379/1" .Release.Name }}
 {{- end }}
 
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "splunk-connect-for-snmp.fullname" -}}
+{{- if .Values.worker.fullnameOverride }}
+{{- .Values.worker.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default (printf "%s-%s" .Chart.Name "user") .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "splunk-connect-for-snmp.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "splunk-connect-for-snmp.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "splunk-connect-for-snmp.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "splunk-connect-for-snmp.labels" -}}
```

Comment:
`splunk-connect-for-snmp.labels` and `splunk-connect-for-snmp.selectorLabels` are only used in https://github.com/splunk/splunk-connect-for-snmp/blob/fix/cleanup-helm-structure-different-traps-and-polling/charts/splunk-connect-for-snmp/templates/serviceaccount.yaml.

Can we directly populate the value there if they are not used?

Reply:
Yes, good idea

Reply:
Or better, I see in every _helpers the labels are the same, like:
```
{{- define "splunk-connect-for-snmp.worker.trap.labels" -}}
helm.sh/chart: {{ include "splunk-connect-for-snmp.worker.chart" . }}
{{ include "splunk-connect-for-snmp.worker.trap.selectorLabels" . }}
{{- if .Chart.AppVersion }}
app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
{{- end }}
app.kubernetes.io/managed-by: {{ .Release.Service }}
{{- end }}
```
Maybe it's better to just use this general labels in every component?

Reply:
Yup, it's better.

Reply:
Make sure that you don't update any existing labels as labels are immutable. (https://banzaicloud.com/blog/helm3-the-good-the-bad-and-the-ugly/#changing-the-unchangeable)

Reply:
Done